_templates/provider/new/package.ejs.t

```diff
@@ -5,7 +5,7 @@
 
 {
   "name": "@novu/<%= name %>",
-  "version": "^0.12.0",
+  "version": "^0.16.0",
```

Comment:
Yesterday I was looking for a way to inject the version in the EJS template dynamically from our package.json version. Didn't find anything yet. 🙁

Reply:
It should just be ` "version": "^0.16.0"` without the `^` otherwise it won't build, that has been my experience